extensions/widgets/mini-profile/src/index.ts

```diff
@@ -8,6 +8,9 @@ export const register: (opts: IntegrationRegistrationOptions) => WidgetInterface
   mountsIn: opts.layoutConfig?.widgetSlotId,
   loadingFn: () => import('./mini-profile-widget'),
   activeWhen: (location, pathToActiveWhen) => {
+    if (location.pathname.includes('/reflection')) {
```

Comment:
```suggestion
 return pathToActiveWhen(['/@akashaorg/app-antenna/reflection', '/@akashaorg/app-antenna/beam'], false)(location)
```

Reply:
the pathToActiveWhen method requires path to be string and not string[]